packages/manager/apps/web/client/app/hosting/multisite/domain-configuration/configuration.controller.js

```diff
@@ -28,6 +28,7 @@ export default class MultisiteDomainConfigurationCtrl {
 
   onCDNChange(isCDNActive) {
     this.cdn = isCDNActive ? CDN_VALUES.ACTIVE : CDN_VALUES.NONE;
+    this.ipv6 = !isCDNActive;
```

Comment:
This doesn't seems to be used. Also the method `onCDNChange` seems to be useless then

Reply:
It's used to display (ipV6: enable) when the cdn is deactivated.

Reply:
it's bind like this: ipv6="selected.domain.ipV6Enabled"

Reply:
Ok seen together